tests/rustdoc/footnote-definition-without-blank-line-100638.rs

```diff
@@ -0,0 +1,15 @@
+#![crate_name = "foo"]
+
+//! Reference to footnotes A[^1], B[^2] and C[^3].
+//!
+//! [^1]: Footnote A.
+//! [^2]: Footnote B.
+//! [^3]: Footnote C.
+
+// @has 'foo/index.html'
+// @has - '//*[@class="docblock"]/*[@class="footnotes"]/ol/li[@id="fn1"]/p' 'Footnote A'
+// @has - '//li[@id="fn1"]/p/a[@href="#fnref1"]' ''
```

Comment:
```suggestion
// @has - '//li[@id="fn1"]/p/a/@href' '#fnref'
```

Might be slightly nicer but whatever

Reply:
Fair enough!